packages/common-utils/src/implementations/BigNumberUtils.ts

```diff
@@ -70,6 +75,22 @@ export class BigNumberUtils implements IBigNumberUtils {
 
     return DecimalString(ethers.formatUnits(valueBigNumber, decimals || 18));
   }
+
+  public BNStoHexString32(bigNumberString: BigNumberString): HexString32 {
+    return HexString32(ethers.toBeHex(this.BNSToBN(bigNumberString)));
+  }
+
+  public BNStoHexString32NoPrefix(
+    bigNumberString: BigNumberString,
+  ): HexString32 {
+    return HexString32(
+      ethers.toBeHex(this.BNSToBN(bigNumberString)).substring(2),
```

Comment:
This is one way to do it. BigInt().toString(16) directly converts to hex without the prefix